src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,65 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    private Movie movie;
+    @OneToOne
+    private CinemaHall cinemaHall;
+    private LocalDateTime showTime;
```

Comment:
Such names should not appear in the database, change them using annotations to this format word_word